### DISCUSSION TITLE
Enabling Istio Ambient mode at cluster level

### DISCUSSION THREAD

**Discussion by sachintiptur** [istio/istio#51759] [Q&A] [answered]
Hi,
We are trying to enable Istio Ambient mode, cluster wide by default, instead of enabling for each namespace. Is there any available installation configuration to achieve this? Something like below which is for side-car mode.
```
# defaults:
  #   sidecarInjectorWebhook:
  #     enableNamespacesByDefault: true
```

**Comment by zirain** [comment]
I recall there's not such configuration now, but sounds good, cc @howardjohn

**Reply by howardjohn** [reply]
Yep. In the past there was but it was removed for the short term until we better understood the use cases

**Reply by sachintiptur** [reply]
Is it still under consideration to add it back?

**Reply by howardjohn** [reply, marked as answer]
It hasn't really been brought up since. the main consideration is what API to expose it under, and if we allow namespaces to opt out, or not, or make that configurable (the idea was we could _enforce_ it is enabled potentially)

**Reply by sachintiptur** [reply]
Ok, then for now we have to enable per Namespace.
